src/tiled/abstracttool.cpp

```diff
@@ -81,21 +83,24 @@ void AbstractTool::setMapDocument(MapDocument *mapDocument)
         return;
 
     if (mMapDocument) {
-        disconnect(mMapDocument, &MapDocument::layerChanged,
-                   this, &AbstractTool::updateEnabledState);
-        disconnect(mMapDocument, &MapDocument::currentLayerChanged,
-                   this, &AbstractTool::updateEnabledState);
+        disconnect(
+            mMapDocument, &MapDocument::layerChanged, this, &AbstractTool::updateEnabledState);
+        disconnect(mMapDocument,
```

Comment:
It seems to be applying the ColumnLimit of 100? I personally use a soft-limit of 80 (that's where I have Qt Creator display a margin line), but it's a soft limit because I always prefer readability to adhering to line length limitations.

Due to explicitly having to name the two classes involved, the connect statements often exceed the line length limit, but I think it's more readable to have it on the same line so I usually keep them that way. In many places you also see them wrapped, which is just because I was more strict about the line length in the past.